.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,62 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines in the [CONTRIBUTING.md](https://github.com/falcosecurity/falco/blob/master/CONTRIBUTING.md) of the main Falco repository.
+2. Please label this pull request according to what type of issue you are addressing.
+3. If the PR is unfinished while opening it specify a wip in the title before the actual title, for example, "wip: my awesome feature"
+-->
+
+**What type of PR is this?**
+
+> Uncomment one (or more) `/kind <>` lines:
+
+> /kind bug
+
+> /kind cleanup
+
+> /kind design
+
+> /kind documentation
+
+> /kind failing-test
+
+> /kind feature
+
+> If this PR will release a new chart version please make sure to also uncomment the following line:
+
+> /kind chart-release
+
+<!--
+Please remove the leading whitespace before the `/kind <>` you uncommented.
+-->
+
+**Any specific area of the project related to this PR?**
+
+> Uncomment one (or more) `/area <>` lines:
+
+> /area charts
+
+> /area proposals
```

Comment:
Not clear to me what area/proposals contains here.

I'd have expected something like 

```
/area falco-chart
/area whatever-chart
```

Reply:
You're right, I'm fixing that.

Reply:
Thanks.

Still not clear to me what `area/proposals` is w.r.t. this repo.

Could you please explain it to me?

Because, IMHO we don't have an area/proposals here...

Reply:
Since the repository structure and all related automation can evolve over time (not just the single), also folks may ask to include a new chart here, I'm assuming that everything not related to a specific pre-existing chart is a proposal.

Maybe I just used the wrong wording?

Reply:
I expect proposals of new charts to go through issues and weekly community calls, before receiving a PR about them.

For this reason, I don't understand why a PR should be labeled `area/proposals` in this case out of nothing.

Anyways, not a big deal. Feel free to do as you prefer. :)

Reply:
Ok, I'm removing it. It's not important now.